demos/python_demos/formula_recognition_demo/README.md

```diff
@@ -25,14 +26,50 @@ Second model is Decoder that takes as input:
 * Target (`tgt`) - previous token (for the first time it is `START_TOKEN` )
 Second model is being executed until current decoded token is `END_TOKEN` or length of the formula is less then `--max_formula_len` producing one token per each decode step.
 
-As input, the demo application takes a path to a folder with images or a path to a single image file with a command-line argument `-i`.
+As input, the demo application takes a path to a folder with images or a path to a single image file with a command-line argument `-i`. Another possible option is interactive mode, which will be explanined in details later.
+> **NOTE**: `--interactive` and `-i/--input` options are mutually exclusive. Please, use one of them, but not both or one of them.
```

Comment:
You could do it with `-i`: if the input is a folder or an image, trigger non-interactive mode, if not and it can be opened with VideoCapture(args.input) or with VideoCapture(int(args.input)) trigger interactive mode. This will also generalize the demo and enable videos as input.

BTW:
```suggestion
> **NOTE**: `--interactive` and `-i/--input` options are mutually exclusive. Please, use one of them, but not both.
```

Reply:
Added interactive mode via `-i` option